packages/Debian/arm64/DEBIAN/control

```diff
@@ -0,0 +1,10 @@
+Package: foglamp-south-pt100
+Version: 0.0
```

Comment:
Seems like we had miss previosuly, It should be `1.0.0` and corresponding change needed for `make_deb`
```
sed -i "s/Version: 1.0.0/Version: ${version}/g" DEBIAN/control

```

Reply:
This does not really matter of course, as the current sed is looking for 0.0 and the Version is set to 0.0.

This is also true of a number of other repositories.

Reply:
I meant for a given a version number `MAJOR.MINOR.PATCH` either be `0.0.0 or 1.0.0`. Anyway will create another JIRA to review semantic version for packaging for repos.

Reply:
Also, if we go for one script in a central place then this will make things easier in future and can resolve this. Only not doing that now to reduce the risk of a change this close to a release.